themes/default/content/case-studies/lykke.md

```diff
@@ -0,0 +1,67 @@
+
+Lykke is now running multiple accounts with many core services each - all defined and managed with Pulumi. Pulumi Crosswalk allows the team to dynamically spin up instances through a single pull request within the stack and manage across multiple environments as needed.
+
+## Results
+
+One of the biggest considerations for the team was cost management. Pulumi helped Lykke adopt cloud-native architectures faster and migrate to managed services like Amazon Elastic Kubernetes Service (EKS) . This helped the Lykke team to run their services more efficiently, reducing costs by at least 20% and making their services more scalable.
+
+Pulumi Crosswalk also helped the team to configure services like Amazon VPC using only a dozen lines of code - something that required 10x more configuration with other IaC tools. The code review process also became faster, with infrastructure changes taking hours, rather than days to deliver.
+
+Within a month of adopting Pulumi, Lykke deployed core supporting infrastructure for a new product. By improving overall productivity for their development team, Lykke was able to deliver the new product to production customers in a matter of months. Lykke is also adopting a comprehensive approach to infrastructure policy using Pulumi’s CrossGuard framework. This ensures that internal security, compliance and cost management policies are automatically enforced through policy-as-code.
+
+## Next steps
+
+Lykke continues to work closely with Pulumi to bring new applications and services to market and deploy new infrastructure capabilities that help their developer team be more productive.
+
+As their next step, the team is reviewing current workloads to optimize existing resources and spin down unused instances that can drive up operating costs. These unaccounted-for resources can easily overwhelm developer budgets and reduce Lykke’s ability to scale their business as they continue to expand their products and attract new customers.
+
+Lykke’s engineering team also plans to adopt unit testing for existing and future infrastructure deployments, as well as explore additional security policies to set up approval and compliance checking for new IAM roles.
+
+## About Pulumi
+
+Pulumi makes building serverless applications easy by automatically provisioning resources and breaking down the development process into standardized building blocks. Check out the following articles to get started:
+
+- [AWS Identity and Access Management with Pulumi Crosswalk](https://www.pulumi.com/docs/guides/crosswalk/aws/iam/)
+- [Amazon Lambda with Pulumi](https://www.pulumi.com/docs/guides/crosswalk/aws/lambda/)
+- [Get Started with Policy as Code and Pulumi CrossGuard](https://www.pulumi.com/docs/get-started/crossguard/)
```

Comment:
```suggestion
- [Get Started with Policy as Code and Pulumi CrossGuard](https://www.pulumi.com/docs/get-started/crossguard/)
- [Get Started with Kubernetes with Pulumi CrossWalk](https://www.pulumi.com/docs/guides/crosswalk/kubernetes/)
```

Reply:
These links should not be fully qualified -- can we make them relative instead?